src/main.js

```diff
@@ -1,11 +1,16 @@
 import toGovspeak from './to-govspeak'
 import insertTextAtCursor from 'insert-text-at-cursor'
+import htmlFromHiddenElement from './html-from-hidden-element'
 
 function htmlFromPasteEvent (event) {
-  if (!event.clipboardData) {
-    return
+  // Modern browsers
+  if (event.clipboardData) {
+    return event.clipboardData.getData('text/html')
+  } else if (window.clipboardData) { // IE propriety property
+    // As IE doesn't support event.clipboardData we have to use a hack to
+    // access HTML from a paste
+    return htmlFromHiddenElement()
```

Comment:
I'm still concerned this is going to be hard to understand for someone coming into this repo for the first time - the question 'why is there a hidden element?' pops up every time I read this, which suggests we could abstract this more cleanly. After reading the file, the hidden element seems to be an implementation detail, so we could just say 'htmlFromPasteEventIE'.